reference/conanfile/attributes.rst

```diff
@@ -1073,6 +1073,7 @@ Properties related to *pkg_config*:
 - **pkg_config_name** property sets the ``names`` property for *pkg_config* generator.
 - **pkg_config_custom_content** property supported by the *pkg_config* generator that will add user
   defined content to the *.pc* files created by this generator
+- **pkg_config_aliases** property sets some aliases of any package/component name for *pkg_config* generator. This property only accepts list-like Python objects.
```

Comment:
Talk with @czoido about this `reference/conanfile/attributest.rst` page, he's refactoring this stuff